src/providers/arcgisrest/qgsnewarcgisrestconnection.cpp

```diff
@@ -63,7 +63,9 @@ QgsNewArcGisRestConnectionDialog::QgsNewArcGisRestConnectionDialog( QWidget *par
     const QString credentialsKey = "qgis/" + mCredentialsBaseKey + '/' + connectionName;
     txtName->setText( connectionName );
     txtUrl->setText( settings.value( key + "/url" ).toString() );
+    Q_NOWARN_DEPRECATED_PUSH
     mHttpHeaders->setFromSettings( settings, key );
+    Q_NOWARN_DEPRECATED_POP
```

Comment:
Why are we using a deprecated method here? Unofficial policy is that if you deprecate something you also remove all the uses of it (unless those uses are from deprecated methods themselves!)

Reply:
This one will be fixed in a follow-up, when the arcgis connection settings are migrated.

The others I believe are meant to stay (either call of deprecated methods in deprecated methods, or the ones used in the compatibility code).